### DISCUSSION TITLE
Supporting more typesetting formats (e.g., AsciiDoc) in the release notes

### DISCUSSION THREAD

**Discussion by vy** [community/community#42860] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

To the best of my knowledge, in its current form, GitHub Release notes only support Markdown, whereas GitHub itself is capable of successfully rendering many other typesetting formats, in particular, including AsciiDoc. If a project has embraced a certain typesetting format (e.g., using AsciiDoc for README and release notes), converting release notes to Markdown becomes a burden. Is it possible to use a different typesetting format than Markdown in GitHub release notes? If not, where/how shall I raise this as a feature request?

**Edit:** Please note that I am talking about **GitHub Release notes**! See step 8 in [Managing releases in a repository](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository).

**Comment by DSDmark** [comment]
GitHub does support rendering other typesetting formats besides Markdown in certain contexts. For example, README files and other file types that are displayed on GitHub can be written in AsciiDoc, reStructuredText, or other formats.

* README files: You can use AsciiDoc, reStructuredText, or other formats to write your project's README file. When you push the file to your repository, GitHub will automatically render it in the appropriate format.

#### AsciiDoc:

```
* Item 1
* Item 2
* Item 3
```
* Item 1
* Item 2
* Item 3

**Reply by vy** [reply]
@DSDmark, I don't think you have read my question right. I already acknowledge that GitHub does support multiple typesetting formats. But the notes in GitHub Releases only support Markdown.
